packages/pancake-uikit/src/components/Svg/index.tsx

```diff
@@ -54,6 +54,7 @@ export { default as PencilIcon } from "./Icons/Pencil";
 export { default as PancakeRoundIcon } from "./Icons/PancakeRound";
 export { default as PocketWatchIcon } from "./Icons/PocketWatch";
 export { default as PlayCircleOutlineIcon } from "./Icons/PlayCircleOutline";
+export { default as PredictionsIcon } from "../../widgets/Menu/icons/Predictions";
```

Comment:
Exporting the Predictions mystic ball thing so that we can access it in the FE

Reply:
Move it to the svg/icons folder then.

Reply:
Yea... we have a weird folder of SVGs in the Menu widget that I think we're agreed should all live here. If I understood him correctly @hachiojidev is gonna look at this while he's doing the menu rework